pkg/util/k8sutil/k8sutil.go

```diff
@@ -315,12 +315,7 @@ func InClusterConfig() (*rest.Config, error) {
 	return rest.InClusterConfig()
 }
 
-func NewTPRClient() (*rest.RESTClient, error) {
```

Comment:
oh, I didn't recognize this design. Not big issue, just looked the code and suspect it can be more elegant.